### PR TITLE
App inprovements, refactoring and documentation 

### PR DESCRIPTION
**Pull request by aviade**

### CODE REVIEW

File: ValidicCSharp/Request/UrlBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿namespace ValidicCSharp.Request
+{
+    using System.Linq;
+
+    public class UrlBuilder
+    {
+        public static string FromCommand(Command command)
```
Comment (gryffindor14) [inline ValidicCSharp/Request/UrlBuilder.cs:7]: Would like to see more unit tests for this class that fail instead of just happy path

Comment (aviade) [reply ValidicCSharp/Request/UrlBuilder.cs:7]: we will be adding more tests, this code is not new, it was moved from the Command class.

**Comment by jkrez** [comment]
Is this repo being actively maintained by Validic?

**Comment by aviade** [comment]
No ones is reviewing this.

**Pull request closed without merging**